hat/assets/js/apps/Iaso/bundle/plugins.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * @param {string} rootDir
+ * @returns {string[]}
+ */
+const getPluginFolders = rootDir => {
+    const pluginsPath = path.resolve(rootDir, '../plugins');
+    return fs.readdirSync(pluginsPath).filter(file => {
```

Comment:
`file` seems like a strange name. If I understand correctly, it should be something like:
```suggestion
    return fs.readdirSync(pluginsPath).filter(pluginName => {
```